Doc/library/sqlite3.rst

```diff
@@ -2614,6 +2590,49 @@ With some adjustments, the above recipe can be adapted to use a
 instead of a :class:`~collections.namedtuple`.
 
 
+.. _sqlite3-howto-encoding:
+
+How to handle non-UTF-8 text encodings
+^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
+
+By default, :mod:`!sqlite3` adapts SQLite values with the ``TEXT`` data type
+using :class:`str`.
+This works well for UTF-8 encoded text, but it will fail for other encodings
+and invalid UTF-8.
+You can use a custom :attr:`~Connection.text_factory` to handle such cases.
+
+Because of SQLite's `flexible typing`_, it is not uncommon to encounter table
+columns with the ``TEXT`` data type, containing non-UTF-8 encodings,
+or even arbitrary data.
+To demonstrate, let's assume we've got a database with ISO-8859-2 (Latin 2)
```

Comment:
```suggestion
To demonstrate, let's assume we have a database with ISO-8859-2 (Latin 2)
```

Reply:
Also, conventional spelling of "Latin-2" is with a hyphen:

```suggestion
To demonstrate, let's assume we have a database with ISO-8859-2 (Latin-2)
```